src/main/java/com/uber/cadence/samples/hello/HelloCron.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.uber.cadence.samples.hello;
+
+import static com.uber.cadence.samples.common.SampleConstants.DOMAIN;
+
+import com.uber.cadence.TerminateWorkflowExecutionRequest;
+import com.uber.cadence.WorkflowExecution;
+import com.uber.cadence.WorkflowIdReusePolicy;
+import com.uber.cadence.activity.Activity;
+import com.uber.cadence.activity.ActivityOptions;
+import com.uber.cadence.client.WorkflowClient;
+import com.uber.cadence.common.CronSchedule;
+import com.uber.cadence.serviceclient.IWorkflowService;
+import com.uber.cadence.serviceclient.WorkflowServiceTChannel;
+import com.uber.cadence.worker.Worker;
+import com.uber.cadence.workflow.Workflow;
+import com.uber.cadence.workflow.WorkflowMethod;
+import java.time.Duration;
+
+/**
+ * Demonstrates a cron workflow that executes activity periodically. Requires a local instance of
+ * Cadence server to be running.
+ */
+public class HelloCron {
+
+  static final String TASK_LIST = "HelloCron";
+  static final String CRON_WORKFLOW_ID = "HelloCron";
+
+  public interface CronWorkflow {
+    @WorkflowMethod(
+      // At most one instance.
+      workflowId = CRON_WORKFLOW_ID,
+      taskList = TASK_LIST,
+      // timeout for every run
+      executionStartToCloseTimeoutSeconds = 30,
+      // To allow starting workflow with the same ID after the previous one has terminated.
+      workflowIdReusePolicy = WorkflowIdReusePolicy.AllowDuplicate
+    )
+    @CronSchedule("*/1 * * * *") // new workflow run every minute
```

Comment:
nit: is 'every min' the least interval we can support for java/go now?

Reply:
No we only support standard cron syntax. Minute is the least interval. 
For seconds cron, use timer and continueAsNew.